.docker/gitpod/Dockerfile

```diff
@@ -2,19 +2,21 @@
 
 # gitpod doesn't support multiple FROM statements, (or rather, you can't copy from one to another)
 # so we just install everything in one go
-FROM gitpod/workspace-base
+FROM ubuntu:jammy
 
 USER root
 
-RUN apt-get update && apt-get install curl git python3 python3-pip python3-venv -y
+RUN apt-get update && apt-get install git curl git -y && apt-get clean
```

Comment:
https://www.gitpod.io/docs/configure/workspaces/workspace-image#custom-base-image suggests we should install `sudo` too.